src/components/Limit/Limit.tsx

```diff
@@ -48,7 +49,7 @@ const Limit: FC<Props> = ({ collection }) => {
     return () => {
       dispatch(setLimit(undefined));
     };
```

Comment:
The `limit` should be cleared when `group_by` do not contains the property used in `order_by`.